Scripts/Editor/SceneLODCreator.cs

```diff
@@ -634,12 +643,17 @@ IEnumerator GetLODMesh(string groupName, Renderer renderer, int depth, Action<Me
                 yield break;
             }
 
-            float quality = Mathf.Pow(groupOptions.VolumePolygonRatio, depth + 1);
+            //if the mesh has under of max triangle count, a max quality should be one
+            //otherwise, it should be a rate of a max triangle.
+            int triangleCount = Math.Max(mesh.triangles.Length / 3, groupOptions.LODTriangleMax);
+            float maxQuality = (float)groupOptions.LODTriangleMax / (float)triangleCount;
+
+            float quality = maxQuality * Mathf.Pow(groupOptions.VolumePolygonRatio, depth + 1);
             int expectTriangleCount = (int) (quality * mesh.triangles.Length) / 3;
 
             //It need for avoid crash when simplificate in Simplygon
             //Mesh has less vertices, it crashed when save prefab.
-            if (expectTriangleCount < 10)
+            if (expectTriangleCount < groupOptions.LODTriangleMin)
```

Comment:
It would be better to create a mesh that fit the minimum number of triangles than using the upper level.